controllers/license/konglicense_controller.go

```diff
@@ -45,16 +45,20 @@ type KongV1Alpha1KongLicenseReconciler struct {
 	LicenseCache     cache.Store
 	CacheSyncTimeout time.Duration
 	StatusQueue      *status.Queue
-	// ControllerName is the name of the controller to use in `controllerName` field in controller status item.
+	// ControllerName is the part of field status.controllers[].controllerName: "LicenseControllerType/ControllerName".
+	LicenseControllerType string
+	// ControllerName is the part of field status.controllers[].controllerName: "LicenseControllerType/ControllerName".
+	// In the KIC, it is set to Election id to use for status update.
+	// REVIEW: Maybe we should rename it to ElectionID?
```

Comment:
👍 You can also make it `mo.Option[string]` so it's clear it's optional.

Reply:
Good idea, I'll change it. But what about the naming?

Reply:
I've changed type, outstanding question about naming

Reply:
Yes, I think that renaming to `ElectionID` makes sense.